hipercow-api/Tools/Utils.cs

```diff
@@ -17,7 +17,16 @@ public static class Utils
         /// <returns>A string which either has a value or is empty, but is not null.</returns>
         public static string HPCString(StoreProperty sp)
         {
-            return sp.Value.ToString() ?? string.Empty;
+            string res = string.Empty;
+            if (sp.Value != null)
+            {
+                if (sp.Value.ToString() != null)
+                {
+                    res = sp.Value.ToString()!;
+                }
+            }
+
+            return res;
```

Comment:
```suggestion
            return sp.Value?.ToString() ?? string.Empty;
```

Reply:
This one is fiddly, because C# thinks `sp.Value.ToString()` can be null so throws me a warning, but in practise I don't think it is possible to actually create a StoreProperty object with with either Value=null or ToString()=null. Code coverage then with the replacement line above gives me "partial coverage" of that line. Expanding out the if seems to satisfy both, but it is very untidy...

I'll see if I can find a better way.

Reply:
Bah, let's just not do code coverage on that one line.